src/python/platforms/android/device.py

```diff
@@ -871,12 +736,9 @@ def push_testcases_to_device():
     logs.log('No testcases to copy to device, skipping.')
     return
 
-  logs.log('Started copying testcases to device.')
```

Comment:
unnecessary log, very unreadable in minimize task with constant back and forth testcase push,